DigitalLearningSolutions.Data/Services/DiagnosticAssessmentDataService.cs

```diff
@@ -25,7 +25,19 @@ public DiagnosticAssessmentDataService(IDbConnection connection, ILogger<Diagnos
         }
 
         public DiagnosticAssessment? GetDiagnosticAssessment(int customisationId, int candidateId, int sectionId)
-        {
+        {
+            // NextTutorialID is the ID of the first tutorial in the section, according to Tutorials.OrderBy
+            // or null if the last in the section. Similar for NextSectionID, using SectionID and SectionNumber
+
+            // Find these by making a list of other tutorials in the course, to then find tutorials in this section,
+            // and other sections (because a section must contain at least one tutorial), using a similar approach to
+            // the one used in the TutorialContentService.
+
+            // Using this list of other tutorials in the course we can work out if there is another item in the

```

Comment:
This kind of comment would probably be useful on the other queries which get similar information about other sections in the course or other items in the section 🙂

Reply:
Yep, I agree! Changed in 7244c04 🙂